proposal-selection.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+---
+
+# Proposal selection process
+
+This process is used when selecting major initiatives to incorporate into the baseline of The Good Docs Project. In particular, it is used when deciding upon proposals to sponsor.
+
+**Version:** 1.0
+
+**Last updated:** August 2020
+
+## When do we use this process?
+
+This process is initiated periodically and typically depends upon:
+
+*   Timing of current build cycles.
+*   Sufficient bandwidth available within the community.
+*   Budget availability.
+
+Timing will typically align with the [Season of Docs](https://developers.google.com/season-of-docs) schedule.
```

Comment:
Does this mean the program itself, or just the time frames for each working phase?
It doesn't make sense (unless I'm missing something) that our project should align proposal work with a completely separate program run by another body. Perhaps the intention with this phrase was more about "We use the SoD as a guide for the different phases of work in a proposal, including how long each might typically run for". Or whatever. And then the URL should probably point to the timeline rather than the home page. No but that won't work because the timeline specifies a date.
I'm tempted to remove this line altogether. If we want proposals to adhere to a certain framework (about timing) then we should create our own and publish that on a separate page.

Reply:
+1 to removing if this line is trying to suggest that we only sponsor projects through Google Season of Docs.

Reply:
Happy to record this as an action but let it release as part of the initial alpha cut of the proposal selection guidelines.

Reply:
Good feedback. Thanks. As discussed in weekly meetings, I've raised this as an issue to address in a future version of processes. #45 

It was considered that this was not a show-stopper for publishing this version.